google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/Firestore.java

```diff
@@ -117,6 +118,19 @@
   ApiFuture<List<DocumentSnapshot>> getAll(
       @Nonnull DocumentReference[] documentReferences, @Nullable FieldMask fieldMask);
 
+  /**
+   * Retrieves multiple documents from Firestore, while optionally applying a field mask to reduce
+   * the amount of data transmitted.
+   *
+   * @param documentReferences Array with Document References to fetch.
+   * @param responseObserver ApiStreamObserver of DocumentSnapshot
+   * @param fieldMask If set, specifies the subset of fields to return.
+   */
+  void getAll(
```

Comment:
Please match the signature above and change the order to:

```getAll(DocumentReference[] documentReferences, @Nullable FieldMask fieldMask,  final ApiStreamObserver<DocumentSnapshot> responseObserver)```

Putting the observer last also makes for much more readable code when a Lambda is used.